Formula/amfora.rb

```diff
@@ -0,0 +1,37 @@
+class Amfora < Formula
+  desc "Fancy terminal browser for the Gemini protocol"
+  homepage "https://github.com/makeworld-the-better-one/amfora"
+  url "https://github.com/makeworld-the-better-one/amfora.git",
+      tag:      "v1.8.0",
+      revision: "71385e9f4e91ab982076856cac95320173ccea73"
+  license "GPL-3.0-or-later"
```

Comment:
Should this be `GPL-3.0-only`? I find no mentions of "or later" in non-template text (see [search query](https://github.com/makeworld-the-better-one/amfora/search?q=or+later)).

Reply:
here is the debian copyright file reference, https://metadata.ftp-master.debian.org/changelogs//main/a/amfora/amfora_1.8.0-1_copyright

Reply:
I think we can go with GPL-3.0-or-later, here is the arch side ref:
![Uploading image.png…]()

Reply:
Debian doesn't indicate `-or-later`, though, and I'm inclined to trust them here. It seems to actually be
```ruby
license all_of: [
  "GPL-3.0-only"
  any_of: ["GPL-3.0-only", "MIT"]
]
```

Reply:
Only [`rr` code](https://github.com/makeworld-the-better-one/amfora/tree/master/rr#license) (amfora's subpackage) is dual-licensed under GPL3 (only?, inherits amfora's license) or MIT. According to this, I think we should stick with `GPL-3.0-only`? 🤔

Reply:
We can specify with comments which licenses apply to which directories/files, like in `e2fsprogs`.

Reply:
> We can specify with comments which licenses apply to which directories/files, like in `e2fsprogs`.

Interesting, thanks for the pointer! 👍

Reply:
> Only [`rr` code](https://github.com/makeworld-the-better-one/amfora/tree/master/rr#license) (amfora's subpackage) is dual-licensed under GPL3 (only?, inherits amfora's license) or MIT. According to this, I think we should stick with `GPL-3.0-only`? 🤔

This is exactly why you can either license it under GPLv3 entirely or both MIT (rr subpackage) and GPLv3 (everything else).

Reply:
we should definitely have some license hell emoji :D